addons/payment/models/account_move.py

```diff
@@ -21,6 +22,13 @@ def _compute_authorized_transaction_ids(self):
                 lambda tx: tx.state == 'authorized'
             )
 
+    @api.depends('transaction_ids')
+    def _compute_total_amount_tx(self):
+        for invoice in self:
+            invoice.total_amount_tx = sum(
+                invoice.transaction_ids.mapped('amount')
```

Comment:
What if there are cancelled/draft transactions ?

(Also, not a big fan of `total_amount_tx` as field name, but I've no inspiration of a good name atm, I suppose amount_paid is already taken?)

Reply:
Good catch! The button wasn't shown anyway because there was a filter on pending, authorized, or done tx.

(I've renamed the field to amount_paid since it wasn't already taken ;) )